### ISSUE TITLE
Proposals should belong to Shows

### ISSUE THREAD

**Issue by subelsky** [ignitebaltimore/ignitespeak#13]
Need a new Show class to keep Proposals organized. A Show object has a title, a date, a time, and a tickets URL.

**Comment by subelsky** [comment]
...and a controller and view for CRUD operations on Shows (Create, Read, Update, Delete)

**Comment by a-jean** [comment]
Hi Mike,

I just want to be sure I understand this story completely. By "Show" are you referring to specific ignite events? 

The goal here is to create a new table in the db with an associated model and controller correct? 

In the show table, do you want a separate field for date and time? Or could I just create a datetime field?

Thanks!

**Comment by subelsky** [comment]
@a-jean Yes, that's right - it's probably more accurate to use the word 'Event' instead of 'Show' - it's a particular date and time where speakers give their presentation

no need to make a separate date and time field because one can easily use SQL functions to extract a date or a time separately when needed. It's more coherent in my view to keep all of that information in one column

**Comment by wistcax** [comment]
@subelsky: I have added the events database. Let me know if that's what you wanted. If there are things you do not like, let me know, thx

**Comment by subelsky** [comment]
@wistcax yes! you're on the right track -- just need to explicitly link the `events` table and the `proposals` table, see my other comments

**Comment by wistcax** [comment]
@subelsky, linking the `events`and `proposals` table is it just adding `has_one` in `proposals`table and `belongs_to` in the `events`table so that that each proposal has only one event? I added it in my repo.
Your other comments were: A) Show object has a title, a date, a time, and a tickets URL, B)  controller and view for CRUD C) no need to make a separate date and time field. To me A),B) & C) are done, I believe. You mean one of these were not done. Thx

**Comment by subelsky** [comment]
OK I think that's right

The proposals table needs to have an `event_id`, which is a non-nullable integer that contains a foreign key reference to an event -- then you'd add `belongs_to :event` in the `Proposal` class, and `has_many :proposals` in the `Event` class

**Comment by wistcax** [comment]
@subelsky I have added the link.
